File: java/src/main/java/model/params/Params.java
package model.params;

public abstract class Params {

    public Params() {
    }

    public Params(String channelType, String symbol) {
        this.channelType = channelType;
        this.symbol = symbol;
    }

    private String channelType;
    private String symbol;

    public String getChannelType() {
        return channelType;
    }

    public void setChannelType(String channelType) {
        this.channelType = channelType;
    }

    public String getSymbol() {
        return symbol;
    }

    public void setSymbol(String symbol) {
        this.symbol = symbol;
    }

    @Override
    public String toString() {
        return "channelType='" + channelType + '\'' +
                ", symbol='" + symbol + '\'';
    }
}

File: c-sharp/ws-client/model/parameters/Params.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    abstract class Params
    {
        protected Params(string channelType, string symbol)
        {
            this.channelType = channelType;
            this.symbol = symbol;
        }

        public string channelType { get; set; }
        public string symbol { get; set; }

    }
}
